doc/syscalls/90001_screen.md

```diff
@@ -30,11 +30,24 @@ The screen driver allows the process to write data to a framebuffer of a screen.
 
     **Returns**: SUCCESS_U32 with 1 if yes, and 0 if no
 
+  * ### Command number: `2`
+
+    **Description**: Set power
+
+    **Argument 1**: 0 if off, nonzero if on.
+
+    **Argument 2**: unused
+
+    **Returns**: Ok(()) followed by the ready callback if the command was successful,
+    BUSY if another command is in progress,
+    INVAL if a brightness value is required but wasn't provided.
+
   * ### Command number: `3`
 
     **Description**: Set brightness
 
-    **Argument 1**: Percent of brightness, 0% should turn off the screen, greater than 0% should turn it on.
+    **Argument 1**: Lightness value, relative to minimum supported.
+    0 should turn off the light if available, greater than 0 should turn it on.
```

Comment:
Yes, percentage is unclear. Brightness could mean either lightness or luminance. Luminance is unhelpful for humans because 50% is not much different from 100%. Lightness is better, but it's closer to a logarithmic relationship, and 0 logarthmiic is not 0 linear, making the linear scale unsuitable.

In addition, percent brings an unnecessary quantization to only 100 levels.

I'd rather leave figuring out the boundaries to the driver. Perhaps specify that the intended range stretches from 1 to u32::max?

Reply:
Leaving the boundaries to the driver would make it pretty hard to write a screen-independent app that sets the screen to max brightness. That said, it may be unrealistic to imagine we will have screen-independent apps with this current interface anyway. Specifying the intended range to some range would be helpful though, I think.

Reply:
Done: I chose >65535 to be the maximum.